interfaces/nav_board.py

```diff
@@ -22,36 +18,35 @@ def __init__(self):
         self._distToGround = 0
         self._lidarQuality = 0  # int 5 for brand new data, counts down 1 every 50ms, should never go below 3.
         self._lastTime = time.time()
-        self.rove_comm_node = core.rovecomm.udp_node
 
-        self.rove_comm_node.subscribe(NAV_IP_ADDRESS)
+        # Set up RoveComm and Logger
+        self.logger = logging.getLogger(__name__)
+
+        core.rovecomm_node.udp_node.subscribe(core.NAV_IP_ADDRESS)
```

Comment:
This would have to be accepted along with NAV_BOARD_IP change
```suggestion
        core.rovecomm_node.udp_node.subscribe(core.NAV_BOARD_IP)
```

Reply:
Going to postpone this change for the same reason as the previous change (future PR addressing all manifest changes)